src/components/TodoItem.js

```diff
@@ -0,0 +1,84 @@
+import { FaTrash } from 'react-icons/fa';
+import { AiFillEdit } from 'react-icons/ai';
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/TodoItem.module.css';
+
+const TodoItem = ({
+  itemProp, setTodos, delTodo, setUpdate,
+}) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleChange = (id) => {
+    setTodos((prevState) => prevState.map((todo) => {
+      if (todo.id === id) {
+        return {
+          ...todo,
+          completed: !todo.completed,
+        };
+      }
+      return todo;
+    }));
+  };
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
+
+  const viewMode = {};
+  const editMode = {};
+  if (editing) {
+    viewMode.display = 'none';
+  } else {
+    editMode.display = 'none';
+  }
+
+  return (
+    <li className={styles.item}>
+      <div className={styles.content} style={viewMode}>
+        <input
+          type="checkbox"
+          checked={itemProp.completed}
+          onChange={() => { handleChange(itemProp.id); }}
+        />
```

Comment:
Please update the style when the item is checked, just like in the tutorial.
Yours 

<img width="590" alt="Screenshot 2023-02-24 at 14 51 30" src="https://user-images.githubusercontent.com/113255559/221195118-a48a45c2-3d21-4a1f-866d-55cebcd80a1d.png">


In the tutorial
<img width="578" alt="Screenshot 2023-02-24 at 14 50 41" src="https://user-images.githubusercontent.com/113255559/221194940-1e1f131f-8d19-44bf-bd24-5ec42aabbd92.png">

Reply:
Hello @Ouail-01 , Thanks for your review, I implemented all the requested changes and will be appreciate if you review my code again, thanks.